packages/sdk/src/dom.ts

```diff
@@ -206,18 +206,40 @@ export function loadScript(
   },
 ) {
   const { attrs = {}, createScriptHook } = info;
-  return new Promise<void>((resolve, _reject) => {
-    const { script, needAttach } = createScript({
-      url,
-      cb: resolve,
-      attrs: {
+
+  const createScriptWithAttrs = (attrs: Record<string, any>) => {
+    return new Promise<void>((resolve, reject) => {
+      const { script, needAttach } = createScript({
+        url,
+        cb: resolve,
+        attrs,
+        createScriptHook,
+        needDeleteScript: true,
+      });
+      needAttach && document.head.appendChild(script);
+      script.onerror = (event) => {
+        script.parentNode?.removeChild(script);
```

Comment:
Should we log the error?
Just to better understand if something different than cors happens

Reply:
Im not sure if it will get spammy, the default setup most configs have do not have CORS headers, in a case like this (like local) it falls back to normal script injection. 

@zhoushaw what are your thoughts?